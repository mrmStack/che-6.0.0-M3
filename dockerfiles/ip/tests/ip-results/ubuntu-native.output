1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN qlen 1
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: ens33: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000
    link/ether 00:0c:29:f4:ca:f6 brd ff:ff:ff:ff:ff:ff
    inet 172.16.148.171/24 brd 172.16.148.255 scope global ens33
       valid_lft forever preferred_lft forever
    inet6 fe80::20c:29ff:fef4:caf6/64 scope link
       valid_lft forever preferred_lft forever
3: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN
    link/ether 02:42:47:f0:7c:bb brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 scope global docker0
       valid_lft forever preferred_lft forever
    inet6 fe80::42:47ff:fef0:7cbb/64 scope link
       valid_lft forever preferred_lft forever
102: br-fe5747c6c290: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN
    link/ether 02:42:72:74:52:0f brd ff:ff:ff:ff:ff:ff
    inet 172.18.0.1/16 scope global br-fe5747c6c290
       valid_lft forever preferred_lft forever
    inet6 fe80::42:72ff:fe74:520f/64 scope link
       valid_lft forever preferred_lft forever
141: br-f2a054118a36: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN
    link/ether 02:42:79:a8:0f:1e brd ff:ff:ff:ff:ff:ff
    inet 172.19.0.1/16 scope global br-f2a054118a36
       valid_lft forever preferred_lft forever
    inet6 fe80::42:79ff:fea8:f1e/64 scope link
       valid_lft forever preferred_lft forever
