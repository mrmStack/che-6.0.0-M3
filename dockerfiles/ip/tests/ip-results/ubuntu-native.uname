4.4.0-21-generic
