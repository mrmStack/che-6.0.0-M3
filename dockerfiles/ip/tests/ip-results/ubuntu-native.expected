172.16.148.171
